server/lib/src/constants/schema.rs

```diff
@@ -1055,4 +1078,16 @@ pub static ref SCHEMA_CLASS_OAUTH2_RS_PUBLIC_DL4: SchemaClass = SchemaClass {
     ..Default::default()
 };
 
+pub static ref SCHEMA_CLASS_APPLICATION: SchemaClass = SchemaClass {
+    uuid: UUID_SCHEMA_CLASS_APPLICATION,
+    name: EntryClass::Application.into(),
+
+    description: "The class representing an application".to_string(),
+    systemmust: vec![Attribute::Name.into()],
+    systemmay: vec![Attribute::Description.into(), Attribute::LinkedGroup.into()],
```

Comment:
systemmust for linked group - an application with no linked group doesnt't make sense.

Reply:
Reading the design doc I interpret two steps, first the application is created, then admin establishes the `app <-> group` association. But we can can change the design document :)

Reply:
I think we need to create the group association as we create the application.